src/atom.rs

```diff
@@ -32,6 +41,38 @@ impl NifAtom {
             false => None
         }
     }
+
+    /// Get the contents of this atom as a string.
+    ///
+    /// (This is slow, and for most cases, unnecessary. For example, if you
+    /// need to know whether or not this atom is `true`, try comparing it to
+    /// `get_atom_init("true")`.)
+    ///
+    pub fn to_string(&self, env: &NifEnv) -> String {
```

Comment:
Sorry I didn't see this last week.

Calling `enif_get_atom_length` is cheap; I'd do that rather than overallocate. You're right about the copy at the end of mine, though -- that's dumb. I'll make a separate PR and try to merge the two.